notebooks/community/feature_store/sdk-feature-store.ipynb

```diff
@@ -0,0 +1,1255 @@
+{
```

Comment:
<ol><li>add a line of basic resource creation for featurestore/entity_type/features resource in the section "In this notebook, you will learn how to:"</li><li>BigQuery not Bigtable</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
added a comment about creating featurestore/entity_type/feature resources